api/models.py

```diff
@@ -109,7 +109,7 @@ class Event(models.Model):
     name = models.CharField(max_length=80)
     description = models.TextField()
     cancelled = models.DateTimeField(null=True, blank=True)
-    winery = models.ForeignKey(Winery, on_delete=models.PROTECT)
+    winery = models.ForeignKey(Winery, on_delete=models.PROTECT, related_name='events')
```

Comment:
nit: this is not really needed, we can use event_set (just learned it.)

Reply:
Interesting. I think I like this approach because it's more explicit, but great catch!